gui/filehandling.py

```diff
@@ -239,16 +239,28 @@ def set_filename(self, value):
 
     filename = property(get_filename, set_filename)
 
-    def init_save_dialog(self):
-        dialog = gtk.FileChooserDialog(
-            C_("Dialogs: Save As...", u"Save"),
-            self.app.drawWindow,
-            gtk.FILE_CHOOSER_ACTION_SAVE,
-            (
-                gtk.STOCK_CANCEL, gtk.RESPONSE_CANCEL,
-                gtk.STOCK_SAVE, gtk.RESPONSE_OK,
-            ),
-        )
+    def init_save_dialog(self, export):
+        if not export:
```

Comment:
Here I might just choose a title in the conditional, and pass it on to the FileChooserDialog constructor in a variable. It would avoid duplicating that long block-style assignment.